how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb

```diff
@@ -640,9 +827,14 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.2"
+   "version": "3.10.5"
+  },
```

Comment:
Can you strip VS Code metadata? -> ideally similar to auto removing pycharm metadata